examples/full/src/components/header/index.js

```diff
@@ -2,6 +2,7 @@ import { h, Component } from 'preact';
 import { Link } from 'preact-router/match';
 import style from './style';
 
+// eslint-disable-next-line
```

Comment:
Added this line because, currently we don't have a `eslint-preact-preset`. 

As of now we are using `eslint-config-synacor`.